messagingpush/src/main/java/io/customer/messagingpush/MessagingPush.kt

```diff
@@ -0,0 +1,111 @@
+package io.customer.messagingpush
+
+import io.customer.messagingpush.api.MessagingPushApi
+import io.customer.messagingpush.data.request.MetricEvent
+import io.customer.messagingpush.di.MessagingPushDiGraph
+import io.customer.messagingpush.hooks.MessagingPushModuleHookProvider
+import io.customer.sdk.CustomerIO
+import io.customer.sdk.CustomerIOInstance
+import io.customer.sdk.di.CustomerIOComponent
+import io.customer.sdk.hooks.HookModule
+import io.customer.sdk.hooks.hooks.ProfileIdentifiedHook
+import io.customer.sdk.repository.PreferenceRepository
+import io.customer.sdk.util.Logger
+
+interface MessagingPushInstance {
```

Comment:
I am not sure about this, so we are saying if a user just wants to add a device to its identified customer and do nothing else, he will still need to add the `messaging` module? 

If they have push notification handling on their own, and just want to track metrics, they won't be able to do that either, without adding the module. 

These basic functionality should just be present in sdk module ?